### ISSUE TITLE
Expected icon at "..." to be NaN pixels wide but was 32.

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-linter#2714]
Regression from #2678, the following manifest.json throws an unexpected warning:

```json
"browser_action": {
    "default_icon": "path/to/valid/file.png",
}
```

`Expected icon at "tests/fixtures/default.png" to be NaN pixels wide but was 32.`

**Comment by vcarciu** [comment]
Verified as fixed using 

```json
"browser_action": {
    "default_icon": "path/to/valid/file.png",
}
```

No warnings were displayed.
